### PR TITLE
Product addition: Signal

### PR DESCRIPTION
**Pull request by Deivedux**
**Type of pull request:** product addition

**Related issues:** no issues

### CODE REVIEW

**Review by doamatto — COMMENTED** [review]
Don't have a chance to check yet, but Signal is already on PrivacySpy ([see here](https://github.com/Politiwatch/privacyspy/blob/master/products/signal.toml))

**Comment by Deivedux** [comment]
Oh wait... It wasn't on a site, how does that happen then?

**Comment by ibarakaiev** [comment]
@Deivedux sorry, that's because we had a year-long PR with legacy import from our old version which we merged just today. I'll still take a look at this PR shortly and add you as a contributor to Signal.

**Comment by Deivedux** [comment]
Thank you for notifying me of this. I'll use the selected version of Signal as a reference to learn how to more accurately read the policies.

**Comment by milesmcc** [comment]
> Thank you for notifying me of this. I'll use the selected version of Signal as a reference to learn how to more accurately read the policies.

To be honest, take it with a grain of salt. There's room for error in these policies. If you see something that looks off or different from your assessment, it's possible that the error is in the other policy—not yours.

**Comment by doamatto** [comment]
We may want to close #66 in favour of this PR; review of this PR coming in a hot minute.

File: products/signal.toml [hunk 1]
```diff
@@ -0,0 +1,58 @@
+name = "Signal"
+description = "A secure messenger with End-to-End Encryption backed by a non-profit."
+slug = "signal"
+hostnames = ["signal.org"]
+sources = ["https://signal.org/legal/"]
```
Comment (doamatto) [inline products/signal.toml:5]: May be a good idea to go straight to the proper section (`https://signal.org/legal/#privacy-policy`)

File: products/signal.toml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+value = "yes"
+notes = ["Signal includes an extensive list of data that it collects, and the reasons why."]
+
+[rubric.data-deletion]
+value = "no"
+notes = ["The policy does not specify a data deletion protocol."]
+
+[rubric.history]
+value = "no"
+notes = ["No history or policy's modification date is made available."]
```
Comment (doamatto) [inline products/signal.toml:26]: Last modified at bottom of page: `Effective as of May 25, 2018 Updated May 25, 2018`. If the site's source is on GitHub, you'd be able to also get a history from there.

File: products/signal.toml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+[rubric.law-enforcement]
+value = "strict"
+citations = ["Other instances where Signal may need to share your data\r\n- To meet any applicable law, regulation, legal process or enforceable governmental request."]
```
Comment (doamatto) [inline products/signal.toml:30]: Could be reduced to `reasonable` with `To protect against harm to the rights, property, or safety of Signal, our users, or the public as required or permitted by law.`

**Review by doamatto — CHANGES_REQUESTED** [review]
Sorry for the long wait for a review. Here's my thoughts.

**Review by doamatto — COMMENTED** [review]
Also, you may just want to use the image already on PrivacySpy; whatever suits your fancy though.

File: products/signal.toml (hunk 1 above)
Comment (milesmcc) [reply products/signal.toml:5]: Also, we typically wouldn't capitalize "End-to-End Encryption" — it's not a proper noun.

**Comment by milesmcc** [comment]
Everything @doamatto says is right. Fix that up and then we should be good to go! Thanks for this contribution.

**Pull request closed without merging**